### PR TITLE
chore: Use string.Method(char) instead of string.Method(string) when possible

### PR DESCRIPTION
**Pull request by Meir017**
ran `dotnet format Garnet.sln --no-restore --verbosity diagnostic` to apply the changes

### CODE REVIEW

File: libs/server/Resp/Bitmap/BitmapCommands.cs [hunk 1]
```diff
@@ -523,14 +523,16 @@ private bool StringBitField<TGarnetApi>(int count, ref TGarnetApi storageApi)
                 }
 
                 //Identify sign for number
+#pragma warning disable CA1865 Use 'string.Method(char)' instead of 'string.Method(string)' for string with single char
                 byte sign = encodingArg.StartsWith("i", StringComparison.OrdinalIgnoreCase) ? (byte)BitFieldSign.SIGNED : (byte)BitFieldSign.UNSIGNED;
```
Comment (Meir017) [inline libs/server/Resp/Bitmap/BitmapCommands.cs:527]: can we use `encoding[0] is 'i' or 'I' ? (byte)BitFieldSign.SIGNED : (byte)BitFieldSign.UNSIGNED;` here?

Comment (badrishc) [reply libs/server/Resp/Bitmap/BitmapCommands.cs:527]: cc @vazois

**Comment by badrishc** [comment]
Warnings in the PR:

```
Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(526,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net6.0]
Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(684,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net6.0]
Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(526,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net8.0]
Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(684,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net8.0]
```

**Comment by Meir017** [comment]
> Warnings in the PR:
> 
> ```
> Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(526,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net6.0]
> Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(684,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net6.0]
> Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(526,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net8.0]
> Warning: /home/runner/work/garnet/garnet/libs/server/Resp/Bitmap/BitmapCommands.cs(684,32): warning CS1696: Single-line comment or end-of-line expected [/home/runner/work/garnet/garnet/libs/server/Garnet.server.csproj::TargetFramework=net8.0]
> ```

I'll wait for us to resolve the above comment (which might remove these directives...)

**Review by badrishc — APPROVED** [review]

**Comment by badrishc** [comment]
could you resolve conflicts in the PR? thanks

**Pull request merged**